modules/local/rename_fastq_cellranger.nf

```diff
@@ -0,0 +1,23 @@
+// Import generic module functions
+process RENAME_FASTQ_CELLRANGER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "conda-forge::python=3.8.0 conda-forge::biopython=1.74"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-adc9bb9edc31eb38b3c24786a83b7dfa530e2bea:47d6d7765d7537847ced7dac873190d164146022-0' :
+        'biocontainers/mulled-v2-adc9bb9edc31eb38b3c24786a83b7dfa530e2bea:47d6d7765d7537847ced7dac873190d164146022-0' }"
+
+    input:
+    tuple val(meta), path(R1), path(R2)
+    tuple val(meta_2), path(orig_r1), path(orig_r2)
+
+    output:
+    tuple val(meta), path('*R1_001.fastq.gz'), path('*R2_001.fastq.gz') , emit: reads
```

Comment:
The files won't necessarily end with `R1_001.fastq.gz` unless we force them to be named this way. Would it be a possibility to rename at this step the fastq files so that they have the name that cellranger expects? We have to be careful with different files from the same sample that correspond to different lanes though